src/driveline_urc/src/MotorCtr/main.cpp

```diff
@@ -0,0 +1,63 @@
+#include <cstdio>
+#include <rclcpp/rclcpp.hpp>
+#include <std_msgs/msg/string.hpp>
+#include <sstream>
+#include "cross_pkg_messages/msg/rover_computer_drive_cmd.hpp"
+#include "geometry_msgs/msg/vector3.hpp"
+#include "CANDriver.h"
+#include "StepperDriver.h"
+#include "DriveTrainMotorManager.h"
+#include "ArmMotorManager.h"
+
+/*
+Note 
+Left stick:
+pitch: pitch
+roll: base rotate
+yaw: M3 rist pitch
+
+right stick:
+pitch: elbow pitch
+roll: rist roll
+yaw: rist/ yaw
+
+left trigger: close end effector
+right trigger: open end effector
+*/
+
+// Callback function
+void callback(const cross_pkg_messages::msg::RoverComputerDriveCMD::SharedPtr msg) {
+   RCLCPP_INFO(rclcpp::get_logger("Motor_CTR"), "Received command with CMD_R.z: %f", msg->cmd_r.z);
+   // wrist_yaw.setVelocity(msg->cmd_r.z);  // Uncomment and set velocity when integrating
+}
+
+int main(int argc, char** argv) {
+    rclcpp::init(argc, argv);
+    auto node = rclcpp::Node::make_shared("motor_ctr");
+
+    RCLCPP_INFO(node->get_logger(), "Motor CTR startup");
+
+    ArmMotorManager arm{};
+
+    // Set loop rate to 800 Hz
+    rclcpp::Rate loop_rate(800);
+
+    // Subscriber for rover drive commands
+    auto driveCommandsSub = node->create_subscription<cross_pkg_messages::msg::RoverComputerDriveCMD>(
+        "/roverDriveCommands", 10, callback);
+
+    // Main loop
+    while (rclcpp::ok()) {
+        rclcpp::spin_some(node);
+
+        arm.tick();  // Handle arm motor manager
```

Comment:
So if I remover correctly, the old version of this executable could either act as a controller for the arm or the driveline, but never at the same time. I believe by switching whether this line or the one below about drive train was commented out switched what function this executable had.

It looks like this new Motor controller executable does not allow for controlling the drivetrain but just the arm.

Since this computer will now only be used for drivetrain control and not arm control, do you think we should remove reference to the arm control? This would include StepperDriver and ArmMOtorManager. Then we can worry about adding them back to the new repo when we get to working on arm.

Reply:
I  agree that since this computer will now only focus on drivetrain control, we should remove references to the arm control, including StepperDriver and ArmMotorManager, to avoid confusion and keep the code clean. This way, we can reintroduce them when we work on the arm. I'll go ahead and remove the arm-related components in this PR.